core/src/test/scala/com/raphtory/enrontest/RaphtoryENRONTest.scala

```diff
@@ -19,26 +19,52 @@ import java.io.File
 @DoNotDiscover
 class RaphtoryENRONTest extends BaseRaphtoryAlgoTest[String] {
 
-  override def batchLoading(): Boolean = false
-
-  override def setSpout(): Spout[String]               =
-    FileSpout("/tmp/email_test.csv")
-  override def setGraphBuilder(): GraphBuilder[String] = new ENRONGraphBuilder()
-
-  override def setup(): Unit =
-    if (!new File("/tmp/email_test.csv").exists())
-      "curl -o /tmp/email_test.csv https://raw.githubusercontent.com/Raphtory/Data/main/email_test.csv " !
+  test("Graph State Test") {
+    val outputFormat: FileOutputFormat = FileOutputFormat(outputDirectory)
 
-  val outputFormat: FileOutputFormat = FileOutputFormat(testDir)
+    graph
+      .liveQuery(graphAlgorithm = GraphState(), outputFormat = outputFormat, increment = 10000)
+      .waitForJob()
 
-  test("Graph State Test") {
-    graph.liveQuery(GraphState(), outputFormat, increment = 10000).waitForJob()
 //    algorithmTest(GraphState(), outputFormat, 1, 32674, 10000, List(500, 1000, 10000))
+
     assert(true)
   }
 
   test("Connected Components Test") {
-    algorithmTest(ConnectedComponents(), outputFormat, 1, 32674, 10000, List(500, 1000, 10000))
+    val outputFormat: FileOutputFormat = FileOutputFormat(outputDirectory)
+
+    val result = algorithmTest(
+            algorithm = ConnectedComponents(),
+            outputFormat = outputFormat,
+            start = 1,
+            end = 32674,
+            increment = 10000,
+            windows = List(500, 1000, 10000)
+    )
+
     assert(true)
```

Comment:
This test is totally broken (not your fault) and needs to be fixed at some point (though that shouldn't block this pull request)